src/containers/Bookings.js

```diff
@@ -13,7 +13,7 @@ export default class Bookings extends Component {
       <div className="App-content">
         <div className="container">
           <Search search={this.search} />
-          {/* <Results results={this.state.results} /> */}
+          <Results />
```

Comment:
As I mentioned in the `Results.js` file we want to pass in the imported fake bookings json with props to the `Results` component to meet the exercise 4 challenge requirements. You can do this with something like `<Results booking={FakeBookings} />